### PR TITLE
Change Werkzeug pinning

### PR DESCRIPTION
**Pull request by ErebusZ**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -50,7 +50,7 @@ install_requires =
     docker
     importlib-metadata; python_version<"3.8"
     jsonschema>=4.4.0
-    protobuf~=3.20
+    protobuf==3.20.1
```
Comment (BlueSquare1) [inline setup.cfg:53]:
```suggestion
    protobuf~=3.20
```
Why not use compatible version like `Werkzeug`

Comment (ErebusZ) [reply setup.cfg:53]: what do you mean compatible version?
also that change is intended.

**Review by benyissa — COMMENTED** [review]
what about setuptools in testing module since it's reverted too?

**Review by elyousfi5 — DISMISSED** [review]

**Comment by ErebusZ** [comment]
> What about setuptools in the testing module since it's been reverted too?

Setuptools was changed by changing the base image in a previous PR.

**Review by 3asm — APPROVED** [review]

**Pull request merged**